core/src/main/scala/io/finch/Input.scala

```diff
@@ -19,4 +35,14 @@ object Input {
   def apply(req: Request): Input = Input(req, req.path.split("/").toList.drop(1))
 
   implicit val inputEq: Eq[Input] = Eq.fromUniversalEquals
+
+  def get(path: String): Input = Input(Request(Method.Get, path))
+  def post(path: String): Input = Input(Request(Method.Post, path))
+  def put(path: String): Input = Input(Request(Method.Put, path))
+  def patch(path: String): Input = Input(Request(Method.Patch, path))
+  def delete(path: String): Input = Input(Request(Method.Delete, path))
+  def connect(path: String): Input = Input(Request(Method.Connect, path))
+  def head(path: String): Input = Input(Request(Method.Head, path))
+  def options(path: String): Input = Input(Request(Method.Options, path))
+  def trace(path: String): Input = Input(Request(Method.Trace, path))
```

Comment:
I don't know if we should support all methods or a restricted set, what do you think?